### PR TITLE
MOSIP-29287 added cloud config for biosdk service qa 115

### PR DESCRIPTION
**Pull request by loganathan-sekaran**

### CODE REVIEW

File: sandbox/biosdk-service-mz.properties [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#iam mock-sdk config
+sdk_check_iso_timestamp_format=true
+mosip.role.biosdk.getservicestatus=REGISTRATION_PROCESSOR
+biosdk_bioapi_impl=io.mosip.mock.sdk.impl.SampleSDKV2
+
+mosip.biosdk.log-request-response-enabled=true
+
+logging.level.root=INFO
+logging.level.io.mosip.biosdk.services.impl.spec_1_0.BioSdkServiceProviderImpl_V_1_0=DEBUG
```
Comment (vishwa-vyom) [inline sandbox/biosdk-service-mz.properties]: Is it requied to keep this part as debug ?

File: sandbox/biosdk-service-mz.properties [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+#iam image-compressor config
+mosip.bio.image.compressor.resize.factor.fx=0.25
+mosip.bio.image.compressor.resize.factor.fy=0.25
+mosip.bio.image.compressor.compression.ratio=50
```
Comment (vishwa-vyom) [inline sandbox/biosdk-service-mz.properties:16]: we need a PR of this first in 1.2.0.1-v3 branch of config to be in sync with release-1.2.0.1 branch of the apps

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: sandbox/biosdk-service-mz.properties (hunk 1 above)
Comment (loganathan-sekaran) [reply sandbox/biosdk-service-mz.properties]: Yes, Currently I want to enable debug logs it to get the request and response of the biosdk API for the testing purpuse. I will disable it later.

File: sandbox/biosdk-service-mz.properties (hunk 2 above)
Comment (loganathan-sekaran) [reply sandbox/biosdk-service-mz.properties:16]: Created PR: https://github.com/mosip/mosip-config/pull/4675

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**